README.md

```diff
@@ -15,43 +16,69 @@ The following setup is designed to only be used when integrating this plugin int
  Add 'mount Hubstats::Engine => "/hubstats"' to your apps routes file.
 
 ## Configuration
-
 ### Authentication
-
-Hubstats needs Github credentials to access your repos, these can be setup in one of two ways:
+Hubstats needs Github credentials to access your repositories, these can be setup in one of two ways:
 
 #### Configuring the `octokit.yml`
-
-Add your GitHub API token or ClientID and Secret to `octokit.yml`.
+Add your GitHub API token (called `access_token`) or ClientID and Secret to `octokit.yml`.
 
 #### Environment Variables
-
 Hubstats can also use OAUTH access tokens stored in ENV["GITHUB_API_TOKEN"] or for Application Authentication in ENV["CLIENT_ID"] and ENV["CLIENT_SECRET"], if for some reason you don't want to store them in `octokit.yml`.
 
 ### Webhooks
-
 Hubstats uses GitHub webhooks to keep itself updated. It requires you to set a secret as well as an endpoint to push to.
 
 To generate a secret run:
-
  ```
  ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'
  ``` 
-
 Set the endpoint to be:
 
  www.yourdomain.com/hubstats/handler
-
+ 
 ### Repositories 
+Hubstats needs to know what repositories for it to watch. You can set it to watch either an entire organization or a list of specific repositories in `octokit.yml`. The list of repositories should look like either:
 
-Hubstats needs to know what repos for it to watch. You can set it to watch either an entire organization or a list of specific repos in octokit.yml.
+```
+repo_list:
+ - sportngin/hubstats
+ - sportngin/active_zuora
+ - sportngin/make-resourceful
+```
 
-## Testing
+or 
 
-All of the automated tests are written in RSpec. Since hubstats is a plugin, not an application, we need to install the plugin into a Rails application to run. The `test` directory is a dummy rails application for manually testing the UI by serving hubstats locally. When developing and using the `test/dummy` locally, then the test will automatically sync with any updated code, so it doesn't need to be re-served when changes are made with the normal Rails application. When in the development process, one just needs to run `bundle exec rails s` once from the command line (while in `test/dummy`) to serve the plugin. http://guides.rubyonrails.org/plugins.html will give more information on the implementation of a Rails plugin. 
+```
+org_name: sportngin
+```
 
-## TL:DR
+### Teams
+Hubstats is automatically set to check a hard-coded list in the `octokit.yml` to know whether a specific organization's team should be watched or not. Because of this, it would be useful to set a list of teams in the `octokit.yml`. The list of teams should look something like:
+
+```
+team_list:
+ - Team One
+ - Team Two
+ - Team Three
+```
+
+Also, the GitHub API token in `octokit.yml` should be a member of all of the teams listed. Sport Ngin's dummy user is `hubstats`. Because of this, there are a couple places where the code takes this dummy user into account, all of which are noted.
```

Comment:
'should be a member' is inaccurate as its required.

Also, remove anything about our specific service user that we use.